tests/features/bootstrap/MinkContext.php

```diff
@@ -164,6 +168,8 @@ public function iOpenTwoBrowserTabsIdentifiedBy($numberOfTabs, $tabNames)
                 'Please identify all tabs you are opening in order to refer to them at a later stage'
             );
         }
+        // Set the testcookie ensuring the selenium tests run in the smoketest env
+        $this->getMink()->getSession()->setCookie('testcookie', 'testcookie');
```

Comment:
is it possible to use more descriptive naming? For example: 'isBehatTest'

Reply:
A good suggestion, but I choose to adhere to the way we called this cookie in OpenConext-engineblock. That way, we have feature parity between the projects.